app/controllers/application_controller.rb

```diff
@@ -6,4 +6,8 @@ class ApplicationController < ActionController::Base
   def hoge(str1)
     str1.start_with?('a') || str1.start_with?('b')
   end
+
+  def command_injection
+     system(params[:command])
```

Comment:
<article><p><b>[Rubocop]</b> <span>Use 2 (not 3) spaces for indentation.</span></p><details><p></p><p>11 line , 3 column in <code>app/controllers/application_controller.rb</code></p><dl><dt>Rule</dt><dd>Layout/IndentationWidth</dd><dt>Severity</dt><dd>convention</dd></dl></details></article>